internal/presenters/presenter_sarif_results_pretty.go

```diff
@@ -84,22 +88,30 @@ Testing %s ...
 %s
 `,
 		meta.TestPath,
-		renderFindings(SortFindings(findings, summary.SeverityOrderAsc)),
+		renderFindings(SortFindings(findings, summary.SeverityOrderAsc), showIgnored, showOpen),
 		renderSummary(summary, meta),
 		getTip(),
 	)
 
 	return str, nil
 }
 
-func renderFindings(findings []Finding) string {
+func renderFindings(findings []Finding, showIgnored bool, showOpen bool) string {
```

Comment:
Issue: Based on the design, if ignored issues are shown, they should get an additional `[ IGNORED ]` indicator.

Reply:
Also it seems that the additional data describing the ignore reason etc is not shown.